lms/static/coffee/src/customwmd.coffee

```diff
@@ -2,10 +2,8 @@
 
 $ ->
 
-  if not MathJax?
-    return
-
-  HUB = MathJax.Hub
+  if MathJax?
+    HUB = MathJax.Hub
```

Comment:
Where is `HUB` used in this file? If `MathJax` is not defined, then `HUB` will not be defined, and anywhere that uses the `HUB` variable will also break.